app/layout.tsx

```diff
@@ -1,12 +1,13 @@
 import './globals.css'
 import type { Metadata } from 'next'
 import { Inter } from 'next/font/google'
+import Navigation from './components/navigation'
 
 const inter = Inter({ subsets: ['latin'] })
 
 export const metadata: Metadata = {
   title: 'Ramani Blog',
-  description: 'Blog posts for Ramani.io',
+  description: 'Generated by create next app',
```

Comment:
I see you have undone this change, could you please keep the description as it is or add more info about the blog